src/nds/maxmod.c

```diff
@@ -0,0 +1,249 @@
+/*
+* This file is part of the OpenSupaplex distribution (https://github.com/sergiou87/open-supaplex).
+* Copyright (c) 2020 Sergio Padrino
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, version 3.
+*
+* This program is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "../audio.h"
+
+#include "../logging.h"
+
+#include "soundbank.h"
+
+#include <maxmod9.h>
```

Comment:
Nit: I prefer this order/agroupation:
```suggestion
#include "../audio.h"
#include "../logging.h"

#include <maxmod9.h>

#include "soundbank.h"
```

From https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes